### PR TITLE
Split it some more functions from canvas-map.js

### PR DESCRIPTION
**Pull request by lukasappelhans**

### CODE REVIEW

File: src/canvas-map.js [hunk 1]
```diff
@@ -69,6 +19,13 @@ function paintBackgroundElement(element, parameters) {
...
 }
 
+function paintStaticElements(elements, parameters) {
+  for (const i in elements) {
+    let element = elements[i];
+    paintStaticElement(element, parameters);
+  }
+}
+
 function extend(extension, obj) {
```
Comment (martgnz) [inline src/canvas-map.js]: Why not moving this file, too?

File: src/canvas-map.js [hunk 2]
```diff
@@ -1,64 +1,14 @@
...
-  const bounds = two;
-  bounds[0][0] = Math.min(one[0][0], two[0][0]);
-  bounds[0][1] = Math.min(one[0][1], two[0][1]);
-  bounds[1][0] = Math.max(one[1][0], two[1][0]);
-  bounds[1][1] = Math.max(one[1][1], two[1][1]);
-  return bounds;
-}
-
-function paintBackgroundElement(element, parameters) {
+function paintStaticElement(element, parameters) {
```
Comment (martgnz) [inline src/canvas-map.js]: paintStaticElement and paintStaticElements could be moved to the util folder?

File: src/util/is-inside-polygon.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+
```
Comment (martgnz) [inline src/util/is-inside-polygon.js]: Delete leading space

File: src/canvas-map.js (hunk 1 above)
Comment (martgnz) [reply src/canvas-map.js]: This review feature is confusing, I am talking about the extend function.

Comment (lukasappelhans) [reply src/canvas-map.js]: Yeah, I thought it about, but haven't finished thinking :) Next PR I think!

File: src/canvas-map.js (hunk 2 above)
Comment (lukasappelhans) [reply src/canvas-map.js]: Yes, I'm not sure about this (plus the already moved `paintFeature`) to be honest. They are very tightly coupled to the canvas map class.

**Pull request merged**